### PR TITLE
Adjust to golangci-lint 1.58.0

### PR DESCRIPTION
**Pull request by tpantelis**

### CODE REVIEW

**Review by aswinsuryan — APPROVED** [review]

File: .golangci.yml [hunk 1]
```diff
@@ -169,5 +169,6 @@ issues:
         - contextcheck
         - errname
         - gochecknoinits
-        - goerr113
+        - init
```
Comment (skitt) [inline .golangci.yml]: Why is this being added?

Comment (tpantelis) [reply .golangci.yml]: To workaround an issue with `gochecknoinits` : https://github.com/golangci/golangci-lint/issues/4697.

Comment (tpantelis) [reply .golangci.yml]: Perhaps I should comment this 😄

Comment (skitt) [reply .golangci.yml]: Could you add a comment to that effect, to avoid head-scratching if this causes errors once golangci-lint is fixed?

**Review by skitt — CHANGES_REQUESTED** [review]

File: .golangci.yml (hunk 1 above)
Comment (tpantelis) [reply .golangci.yml]: just did

Comment (skitt) [reply .golangci.yml]: Hah I hadn’t seen your comment!

**Review by skitt — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**